10/alpine/Dockerfile

```diff
@@ -37,10 +42,14 @@ RUN addgroup -g 1000 node \
     && gpg --batch --decrypt --output SHASUMS256.txt SHASUMS256.txt.asc \
     && grep " node-v$NODE_VERSION.tar.xz\$" SHASUMS256.txt | sha256sum -c - \
     && tar -xf "node-v$NODE_VERSION.tar.xz" \
+    && ln -s /usr/bin/ccache /usr/local/bin/gcc \
+    && ln -s /usr/bin/ccache /usr/local/bin/g++ \
     && cd "node-v$NODE_VERSION" \
     && ./configure \
     && make -j$(getconf _NPROCESSORS_ONLN) \
     && make install \
+    && rm /usr/local/bin/gcc \
+    && rm /usr/local/bin/g++ \
```

Comment:
Should probably also include an `rm -rf /root/.ccache` too

Reply:
Yeah I'm working on pushing and cleaning up after.

Which bring a question. I need to do this in bash and I am not sure how (below is pseudo code)

If access token exist
  git add
  git commit
  git push
end if
rm -rf .ccache

I'm not sure how to chain it. Do I need to create a subshell with `$( )`?

/cc @PeterDaveHello